getting_started/scripting/c_sharp/c_sharp_features.rst

```diff
@@ -222,7 +222,7 @@ Full list of defines
 
 * One of ``GODOT_64`` or ``GODOT_32`` is defined depending on if the architecture is 64-bit or 32-bit.
 
-* One of ``GODOT_X11``, ``GODOT_WINDOWS``, ``GODOT_OSX``, ``GODOT_ANDROID``, ``GODOT_HTML5``,
+* One of ``GODOT_LINUXBSD``, ``GODOT_WINDOWS``, ``GODOT_OSX``, ``GODOT_ANDROID``, ``GODOT_HTML5``,
```

Comment:
Is this the correct name of the Mono define?

cc @aaronfranke

Reply:
Very likely, but I can't check because it doesn't build anymore.